ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -72,6 +72,30 @@
                     window.alert("Issue while saving draft");
                 }
             }, "saveError");
+            const autoSaveProperties = params?._formJson?.properties?.['fd:autoSave'];
+            this.#setupAutoSave(self.getModel(), autoSaveProperties);
```

Comment:
AutoSaveProperties should be part of formModel. They shouldn't be required to pass separately.

Reply:
Done